### PR TITLE
31 add ydata profiling documentation

### PR DESCRIPTION
**Pull request by Arnwald**
New documentation page for ydata-profiling service.

### CODE REVIEW

File: docs/reference/services/ydata-profiling.md
Comment (ludelafo) [inline docs/reference/services/ydata-profiling.md:1]: The structure of this file has changed a bit to refactor common elements to the [Core concepts > Service](https://docs.swiss-ai-center.ch/reference/core-concepts/service/) reference.

You can check an existing service to check how to adapt yours to the updated version. This mostly concerns the removing of some parts of your current documentation and pointing to the reference mentioned above.

**Review by ludelafo — CHANGES_REQUESTED** [review]
You might need to pull the changes from main to get all new commits and be synced with the main branch.

File: docs/reference/index.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ mdwrap is a command line tool to format our Markdown documentation made with
 | [nlp-langid](./services/nlp-langid.md)                   | <https://github.com/swiss-ai-center/nlp-langid-service>            | <https://nlp-langid-swiss-ai-center.kube.isc.heia-fr.ch>            | -              |
 | [text-summarizer](./services/text-summarizer.md)         | <https://github.com/swiss-ai-center/text-summarizer-service>       | <https://text-summarizer-swiss-ai-center.kube.isc.heia-fr.ch>       | -              |
 | [yolov8](./services/yolov8.md)                           | <https://github.com/swiss-ai-center/yolov8-service>                | <https://yolov8-swiss-ai-center.kube.isc.heia-fr.ch>                | -              |
+| [yolov8](./services/ydata-profiling.md)                  | <https://github.com/swiss-ai-center/ydata-profiling>               | <https://ydata-profiling-swiss-ai-center.kube.isc.heia-fr.ch>       | -              |
```
Comment (andrptrc) [inline docs/reference/index.md:58]: The service is already in the table. Just above

File: mkdocs.yml
Comment (andrptrc) [inline mkdocs.yml:1]: Not sure about the indent here, try to run the documentation-format container to be sure.

**Review by andrptrc — CHANGES_REQUESTED** [review]

File: mkdocs.yml
Comment (Arnwald) [reply mkdocs.yml:1]: I just tested it and it runs fine.

File: docs/reference/services/ydata-profiling.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# ydata-profiling
+
+- [:material-account-group: Main author - HES-SO Valais-Wallis](https://www.hes-so.ch/swiss-ai-center/equipe)
+- [:material-git: Code](https://github.com/swiss-ai-center/ydata-profiling-service/)
+- [:material-kubernetes: Deployment configuration](https://github.com/swiss-ai-center/ydata-profiling-service/tree/main/kubernetes)
+- [:material-test-tube: Staging](https://ydata-profiling-swiss-ai-center.kube.isc.heia-fr.ch)
+- [:material-factory: Production (not available yet)](https://ydata-profiling.swiss-ai-center.ch)
+
```
Comment (andrptrc) [inline docs/reference/services/ydata-profiling.md:8]: I would add "Description" title here to be in the same format as other services

**Review by andrptrc — CHANGES_REQUESTED** [review]

**Review by andrptrc — APPROVED** [review]

**Review by ludelafo — APPROVED** [review]

**Pull request merged**